### ISSUE TITLE
Character causes crash when saving

### ISSUE THREAD

**Issue by clausimu** [pioneerspacesim/pioneer#4270]
### Observed behaviour
I'm working on persistent entities (corporations, groups, etc.) with persistent characters and ships that even exist outside of the player system (for storyline and more missions). As discussed on IRC, one problem I'm having is that the game crashes when saving (even though the script is currently not serializing anything). 
I was able to pinpoint the problem to the following circumstance:

1) An entity is creating a ship.
2) The crew for the ship is handed from the entity (script file "entity.lua") to the ship (script file "starship.lua"). If the Characters for the crew are created within the "starship.lua" script file, saving works just fine.

### Expected behaviour
No crash when saving.

### Steps to reproduce
Just place the two attached files into a folder in /modules. They are a very bare-bones version just for reproducing the bug. Load game in Epsilon Eridani (should work anywhere, but that is usually where I test). A new ship will be created automatically within 100 km. Save game. Booom!

[SaveBug.zip](https://github.com/pioneerspacesim/pioneer/files/1617071/SaveBug.zip)

__My pioneer version (and OS):__
pioneer version: git build from 1/3/18
OS: openSUSE 43.3

**Comment by jaj22** [comment]
Typo plus optional parameters. entity.lua uses `Character:New()`, but the function spec is new = function(template, rand), so it passes the Character table in as a template, with unwanted results. New() explicitly prevents using characters as templates, but not its own parent table.

`Character.New()` appears to work fine.

**Comment by clausimu** [comment]
@jaj22 Thank you! That was it. I think I even understand why I wrote it like that. In the real script I pass a rand object to the Character.New function. That crashes (because it tries to interpret rand as template). Instead of just feeding an empty table (=template) and rand, I changed it to Character:New(rand). That has been my first-try-solution when function calls don't work because I usually get the function.call() and function:call() mixed up. As you said - both work with Character.New(), but with bad and unintended consequences...

Long story short - Thank you!
